src/main/java/se/iths/springbootgroupproject/services/MessageService.java

```diff
@@ -15,6 +16,26 @@ public class MessageService {
     public MessageService(MessageRepository messageRepository) {
         this.messageRepository = messageRepository;
     }
+    public Message updateMessage(Long id, Message updateMessage) {
+        Message existingMessage = messageRepository.findById(id).orElseThrow(()-> new EntityNotFoundException("Message not found with id: "+id));
+        if (updateMessage.getDate() != null) {
+            existingMessage.setDate(updateMessage.getDate());
+        }
```

Comment:
can we replace this with    existingMessage.setDate(LocalDate.now()); so that the date automatically is updated when updating a message?

Reply:
I think this could be done in another way, with annotations. Please see issue #25.

Reply:
could you explain a little more? Is it enough to annotate the field "date" with @LastModifiedDate instead of @CreatedDate?

Reply:
I think we would need another field in the message entity, e.g. lastModifiedAt, and then use the @LastModifiedDate annotation on that field.